### PR TITLE
[onert-micro] Move kernels list to rodata section

### PR DESCRIPTION
**Pull request by BalyshevArtem**
This pr replaces kernels list to rodata.

issue: #10686

ONE-DCO-1.0-Signed-off-by: Artem Balyshev <a.balyshev@samsung.com>

### CODE REVIEW

File: onert-micro/CMakeLists.txt [hunk 1]
```diff
@@ -93,7 +93,7 @@ if (GENERATE_KERNELS_LIST_FROM)
     list(APPEND CMAKE_ARM_OPTIONS "-DLUCI_INTERPRETER_KERNELS_BUILD_LIST=${GENERATED_KERNELS_LIST_PATH}")
 endif ()
 
-if (ENABLE_TEST)
+if (ENABLE_ONERT_MICRO_TEST)
     message(STATUS "Enable tests")
```
Comment (BalyshevArtem) [inline onert-micro/CMakeLists.txt:97]: Small fix, to rename variable due to `ENABLE_TEST` already exist

File: onert-micro/luci-interpreter/src/kernels/KernelBuilder.h [hunk 2]
```diff
@@ -19,71 +19,133 @@
...
   }
+
+private:
+  KernelExecuteFunc *_operator_execute[size_t(BuilderID::Size)];
 };
 
+// Global constexpr kernel configure and kernel executor
+constexpr KernelConfigureRegistry kernel_configure;
+constexpr KernelExecuteRegistry kernel_executor;
+
```
Comment (BalyshevArtem) [inline onert-micro/luci-interpreter/src/kernels/KernelBuilder.h:148]: Create constexpr KernelConfigureRegistry and KernelExecuteRegistry, so now they can save in rodata

File: onert-micro/luci-interpreter/pal/mcu/pal.cmake [hunk 3]
```diff
@@ -29,13 +29,12 @@ macro(initialize_pal)
...
 
 macro(add_pal_to_target TGT)
-    target_include_directories(${TGT} PRIVATE "${PAL}")
     target_include_directories(${TGT} PRIVATE
             "${TensorFlowRuySource_DIR}"
             "${TensorFlowGEMMLowpSource_DIR}"
             "${TensorFlowEigenSource_DIR}"
             "${TensorFlowSource_DIR}")
-    target_include_directories(${TGT} PRIVATE ${LUCI_INTERPRETER_PAL_DIR})
+    target_include_directories(${TGT} PUBLIC ${LUCI_INTERPRETER_PAL_DIR})
```
Comment (chunseoklee) [inline onert-micro/luci-interpreter/pal/mcu/pal.cmake:37]: Why this PR need this ?

**Review by chunseoklee — APPROVED** [review]

File: onert-micro/luci-interpreter/pal/mcu/pal.cmake (hunk 3 above)
Comment (BalyshevArtem) [reply onert-micro/luci-interpreter/pal/mcu/pal.cmake:37]: It resolves some build issues with dependencies

**Pull request merged**